### PR TITLE
[test] Add test for issue6229

### PR DESCRIPTION
**Pull request by wanghongjuan**
- Add test for isseu #6229
- This case is failed on failed version 0.26.0, pass on fixed version 0.27.4

### CODE REVIEW

File: test/sanity/issue6229-mac-executeScript/test.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import time
+import os
+import platform
+import sys
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from nw_util import *
+
+if platform.system() != 'Darwin':
```
Comment (rogerwang) [inline test/sanity/issue6229-mac-executeScript/test.py]: Please enable this case for all platforms, although it was reported for mac but it applies to all. And please rename the case as well.

Comment (wanghongjuan) [reply test/sanity/issue6229-mac-executeScript/test.py]: Updated and tested on Windows/Linux with v0.26.0 and v0.27.4, got same result as Mac OS, "issue6229-mac-executeScript" renamed to "issue6229-webview-executeScript"

**Comment by rogerwang** [comment]
It failed on all build slaves. Please fix

**Comment by rogerwang** [comment]
Traceback (most recent call last):

  File "E:\build\nw28_sdk_win32\node-webkit\src\content\nw\test\sanity\issue6229-webview-executeScript/test.py", line 24, in <module>

    driver.switch_to.window(handles[0])

AttributeError: 'WebDriver' object has no attribute 'switch_to'

Command: C:\Python27\Scripts\..\python E:\build\nw28_sdk_win32\node-webkit\src\content\nw\test\sanity\issue6229-webview-executeScript/test.py

**Comment by GnorTech** [comment]
reverted for now

**Comment by wanghongjuan** [comment]
Submitted a new PR #6478 to fix this issue.

**Pull request merged**